ui/components/imageWorkspace/src/components/Features/TextInputFeature.svelte

```diff
@@ -18,30 +18,53 @@
 
   import { Input } from "@pixano/core/src";
 
-  import type { TextFeature, NumberFeature } from "../../lib/types/imageWorkspaceTypes";
+  import type { TextFeature, IntFeature, FloatFeature } from "../../lib/types/imageWorkspaceTypes";
 
-  export let textFeature: Pick<NumberFeature | TextFeature, "name" | "value">;
+  export let textFeature: Pick<IntFeature | FloatFeature | TextFeature, "name" | "value">;
   export let isEditing: boolean;
   export let saveInputChange: (value: string | number, propertyName: string) => void;
-  export let inputType: "text" | "number" = "text";
+  export let inputType: "str" | "int" | "float" = "str";
 
   let isSaved = false;
 
   const onTextInputChange = (value: string, propertyName: string) => {
-    const formattedValue = inputType === "number" ? Number(value) : value;
+    let formattedValue: string | number = value;
+    if (inputType === "int") {
+      formattedValue = Math.round(Number(value));
+    } else if (inputType === "float") {
+      formattedValue = Number(value);
+    }
     saveInputChange(formattedValue, propertyName);
     isSaved = true;
   };
 </script>
 
 <div class="flex justify-start items-center gap-4">
   {#if isEditing}
-    <Input
-      value={textFeature.value}
-      on:change={(e) => onTextInputChange(e.currentTarget.value, textFeature.name)}
-      on:input={() => (isSaved = false)}
-      type={inputType}
-    />
+    {#if inputType === "str"}
```

Comment:
Here is as suggested simplification : 
```
<Input
      value={textFeature.value}
      on:change={(e) => onTextInputChange(e.currentTarget.value, textFeature.name)}
      on:input={() => (isSaved = false)}
      type={inputType === "str" ? "text" : "number"}
      step={inputType === "int" ? "1" : "any"} 
    />
```

`step=any` would not change anything if input type is text !